index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h1>My Shopping Webpage</h1>
+    </header>
+
+    <nav>
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">Products</a></li>
+        <li><a href="#">Cart</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section>
+      <h2>Featured Products</h2>
+      <div class="product">
+        <img src="product1.jpg" alt="Product 1" />
```

Comment:
use valid image from local folder or a working link

Reply:
- [ ] It is important that for every comment you add you should as well go deep by explaining the `why` a change is being requested. This would assist the students to better understand the reason for a change and they know how best to go about it in their next projects :+1: :sparkles: 
- [ ] please i kindly suggest you try to sound polite words when requesting for changes from the student, you can use words such as `i kindly suggest, i think you can, please ensure you` :+1: :sparkles: